### PR TITLE
feat(grpc-process)

### PR DESCRIPTION
**Pull request by kafkaphoenix**
## Description

- Add grpc trigger (python and go)
- Add __init__.py to python processes

## Motivation and Context

grpc trigger enables users to have a predefined trigger that allows grpc based operations.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by alexrodfe — APPROVED** [review]

File: grpc-trigger/app.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  version_id: GRPC Trigger version v1.0.0
+  base_path: "/Some/Path"
+
+nats:
+  url: "nats://localhost:4222"
+  stream: trigger
+  output: grpc-trigger-output
+
+centralized_configuration:
+  product:
```
Comment (dfinteligenz) [inline grpc-trigger/app.yaml:14]: You will need a global scope too.

**Review by dfinteligenz — APPROVED** [review]

File: grpc-trigger/app.yaml (hunk 1 above)
Comment (kafkaphoenix) [reply grpc-trigger/app.yaml:16]: updated

**Pull request merged**